war/src/main/fonts/font-awesome/css/font-awesome.css

```diff
@@ -1,6 +1,6 @@
 /*!
- *  Font Awesome 4.6.2 by @davegandy - http://fontawesome.io - @fontawesome
- *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
+ *  Font Awesome 4.6.2 by @davegandy - https://fontawesome.com - @fontawesome
+ *  License - https://fontawesome.com/license (Font: SIL OFL 1.1, CSS: MIT License)
```

Comment:
Given this is likely a local copy of font-awesome CSS file should this be changed from the original?

Reply:
Folks have been submitting typo fixes in our YUI debug files for a looooong time and they've all been approved...